tests/common/helpers/general.py

```diff
@@ -0,0 +1,20 @@
+from netaddr import IPNetwork
+
+def generate_ips(num, prefix, exclude_ips):
+    """ Generate random ips within prefix """
+    prefix = IPNetwork(prefix)
+    exclude_ips.append(prefix.broadcast)
+    exclude_ips.append(prefix.network)
+    available_ips = list(prefix)
```

Comment:
Optional: The netaddr package has class IPSet which serves this purpose better.

Reply:
For IPSet usage there is still need to do manipulation like not to include specific IP addresses.
So prefer to keep current implementation.